e2e/cypress/support/api_commands.js

```diff
@@ -482,13 +563,12 @@ Cypress.Commands.add('apiUpdateConfig', (newSettings = {}) => {
             body: settings,
         });
     });
-
-    cy.apiLogout();
 });
 
 Cypress.Commands.add('apiGetConfig', () => {
-    cy.apiLogin('sysadmin');
-
     // # Get current settings
-    return cy.request('/api/v4/config');
+    return cy.request('/api/v4/config').then((response) => {
+        expect(response.status).to.match(/20\d/);
```

Comment:
I think it's convenient to match status like `/20\d/`.  However, I think we should be more explicit.

Reply:
Will do!